### DISCUSSION TITLE
Gov cloud support exists

### DISCUSSION THREAD

**Discussion by shubhamrajvanshi** [kubernetes-sigs/azuredisk-csi-driver#2112] [General]
Can this driver be used in the Azure gov environment?

**Comment by andyzhangx** [comment]
the aks managed csi driver supports Azure gov environment

**Reply by shubhamrajvanshi** [reply]
Is it available for OSS helm chart

**Reply by andyzhangx** [reply]
it should support as long as you have configured the azure cloud config, check here for details: https://github.com/kubernetes-sigs/azuredisk-csi-driver#prerequisite

**Comment by shubhamrajvanshi** [comment]
Thanks @andyzhangx I can confirm it works as pointed. I was wondering if the key for the secret can be provided by user instead of hard-coded as "cloud-config". I can take this up on my own, willing to contribute.

**Reply by andyzhangx** [reply]
@shubhamrajvanshi if you are using aks, this driver could rely on `/etc/kubernetes/azure.json` file on the agent node to get managed identity auth
